cmd/frontend/graphqlbackend/site_alerts.go

```diff
@@ -232,6 +232,11 @@ func outOfDateAlert(args AlertFuncArgs) []*Alert {
 
 // This should be removed from 3.27
 func deprecationAlert(args AlertFuncArgs) []*Alert {
+	sourcegraph_dotcom, _ := strconv.ParseBool(env.Get("SOURECGRAPHDOTCOM_MODE", "false", "determines Sourcegraph environment type"))
+	if sourcegraph_dotcom == true {
```

Comment:
```suggestion
	if envvar.SourcegraphDotComMode() { {
```

and updating the imports :) 

(FYI [I just searched for "dotcom" in "graphqlbackend" and found this](https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+dotcom+file:graphqlbackend&patternType=literal))

Reply:
Noice!